app/lib/features/chat/pages/edit_chat_page.dart

```diff
@@ -0,0 +1,218 @@
+              const SizedBox(height: 40),
+              _buildAvatarUI(),
+              const SizedBox(height: 40),
+              _chatName(),
+              const SizedBox(height: 20),
+              _chatDescription(),
+              const SizedBox(height: 30),
+              _saveButton(),
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+
+  Future<void> pickAvtar() async {
+    FilePickerResult? result = await FilePicker.platform.pickFiles(
+      dialogTitle: L10n.of(context).uploadAvatar,
+      type: FileType.image,
+    );
+    if (result != null && result.files.isNotEmpty) {
+      selectedUserAvatar.value = File(result.files.first.path!);
+    }
+  }
+
+  Widget _buildAvatarUI() {
+    return ValueListenableBuilder(
+      valueListenable: selectedUserAvatar,
+      builder: (context, userAvatar, child) {
+        return GestureDetector(
+          onTap: () => pickAvtar(),
+          child: Container(
+            height: 100,
+            width: 100,
+            decoration: BoxDecoration(
+              image: userAvatar != null
+                  ? DecorationImage(
+                      image: FileImage(userAvatar),
+                      fit: BoxFit.cover,
+                    )
+                  : null,
+              color: Theme.of(context).colorScheme.primaryContainer,
+              borderRadius: BorderRadius.circular(5),
+            ),
+            child: userAvatar == null
+                ? Icon(
+                    Atlas.up_arrow_from_bracket_thin,
+                    color: Theme.of(context).colorScheme.neutral4,
+                  )
+                : null,
+          ),
+        );
+      },
+    );
+  }
+
+  Widget _chatName() {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: <Widget>[
+        Padding(
+          padding: const EdgeInsets.only(bottom: 5),
+          child: Text(L10n.of(context).chatName),
+        ),
+        InputTextField(
+          hintText: L10n.of(context).typeName,
+          textInputType: TextInputType.multiline,
+          controller: chatNameController,
+        ),
+        const SizedBox(height: 3),
+        Text(
+          L10n.of(context).egGlobalMovement,
+          style: Theme.of(context).textTheme.labelSmall!.copyWith(
+                color: Theme.of(context).colorScheme.neutral4,
+              ),
+        ),
+      ],
+    );
+  }
+
+  Widget _chatDescription() {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: <Widget>[
+        Padding(
+          padding: const EdgeInsets.only(bottom: 5),
+          child: Text(L10n.of(context).about),
+        ),
+        InputTextField(
+          hintText: L10n.of(context).description,
+          textInputType: TextInputType.multiline,
+          controller: chatDescriptionController,
+          maxLines: 10,
+        ),
+      ],
+    );
+  }
+
+  Future<void> _save() async {
+    if (chatNameController.text.trim().isEmpty) return;
```

Comment:
yeah, no ... you can set the description and name independently of one another ... forcing a name and always updating both isn't quite right...